src/Core/Models/Data/DisableSendPolicyData.cs

```diff
@@ -0,0 +1,9 @@
+using Bit.Core.Enums;
+
+namespace Bit.Core.Models.Data
+{
+    public class DisableSendPolicyData
+    {
+        public DisableSendPolicyType? DisableSend { get; set; }
```

Comment:
Remove this property. Instead just "`public bool DenyHiddenEmail { get; set; }`"